articles/azure/azs-how-create-vm-terraform.md

```diff
@@ -374,14 +389,14 @@ resource "azurestack_virtual_machine" "test" {
     environment = "staging"
   }
 }
-```
+</code></pre>
 
 ## How to execute a plan - using **Environment Variables** option
 
 > [!TIP]
-> Terraform by default scans your execution directory and is looking for all `tf` files.
+> Terraform by default scans your execution directory and is looks for all `tf` files.
```

Comment:
I think you need to remove the word "is" for the sentence to make sense:

Terraform, by default, scans your execution directory and looks for all `tf` files.

Reply:
Fixed